testserver/manage.py

```diff
@@ -3,6 +3,15 @@
 import sys
 
 if __name__ == "__main__":
+    
+    if 'python3' not in sys.executable:  # might be better to check version also.
```

Comment:
Good point; I had not arrived at that solution; but I like it better. Implemented.